docs/sources/developers/plugins/share-a-plugin.md

```diff
@@ -1,120 +1,69 @@
 
 When you build a plugin from source, a `dist` directory is created that contains the production build, or _plugin assets_, for your plugin.
 
-When loading your plugin, Grafana only cares about the plugin assets. Specifically, when the Grafana server starts, it attempts to discover and load plugins like this:
+When loading your plugin, Grafana only cares about the plugin assets. Specifically, when the Grafana server starts, it attempts to discover and load plugins in the following order:
 
 1. Look for a `plugin.json` file in any of the subdirectories in the plugin directory.
 1. If a `plugin.json` was found, try to load the plugin assets from a `dist` directory in the same directory as the `plugin.json` file.
 1. If there's no `dist` directory, try to load the plugin assets from the same directory as the `plugin.json` file.
 
-Now that you know what Grafana needs to load your plugin, let's see how you can share the plugin with other users.
+## Package the plugin
 
-The best way to share your plugin with the world is to publish it on [Grafana Plugins](https://grafana.com/plugins). However, if you're not ready to make your plugin public just yet, you can still share your plugin by hosting the plugin yourself.
+Now that you know what Grafana needs to load your plugin, let's see how you can package and share the plugin with other users.
 
-## Publish your plugin on Grafana.com
-
-To publish a plugin to [Grafana Plugins](https://grafana.com/grafana/plugins), your plugin first needs to be publicly available in a commit on [GitHub](https://github.com).
-
-The commit you submit needs to either:
-
-- Contain a `dist` directory with the plugin assets
-- Contain the plugin assets in the root directory
+### 1. Build the plugin
 
-We strongly recommend that you don't check in the plugin assets to the main branch. Instead, use the following steps to create a release branch that contains the plugin assets.
-
-1. Create a release branch.
+1. Install dependencies.
 
    ```
-   git checkout -b release-0.1.x
+   yarn install --pure-lockfile
    ```
 
-1. Build the plugin assets.
+1. Build plugin assets.
 
    ```
    yarn build
    ```
 
-1. Add the `dist` directory. The `-f` flag adds the directory even if it's ignored by `.gitignore`.
-
-   ```
-   git add -f dist
-   ```
-
-1. Create the release commit.
+1. (Optional) If your data source plugin has a backend plugin, build it as well.
 
    ```
-   git commit -m "Release v0.1.0"
+   mage
    ```
 
-1. Create a release tag. You can also [create the release on GitHub](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository). If you do, then you can skip this step and the next one.
+### 2. Sign the plugin
 
-   ```
-   git tag -a v0.1.0 -m "Create release tag v0.1.0"
-   ```
+1. [Create an account on Grafana.com](https://grafana.com/signup) and generate an API key with the `PluginPublisher` role.
```

Comment:
This section should explain the differences between the signature types, and that the user can contact us to create community or commerical signed plugins.  It should also mention the --rootUrls parameter for private signatures and how to use it.

Reply:
I see the signature types are mentioned below, but unless they are either registered to make public signatures or provide rootUrls for a private signature, these instructions won't work as they're written today.